### PR TITLE
Update gems with bummr

### PR DESCRIPTION
**Pull request by monfresh**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -154,7 +154,6 @@ Style/IfUnlessModifier:
   Description: Favor modifier if/unless usage when you have a single-line body.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier
   Enabled: true
-  MaxLineLength: 100
```
Comment (tbaxter-18f) [inline .rubocop.yml]: Why was this removed?

Comment (monfresh) [reply .rubocop.yml]: It has been deprecated by Rubocop. This cop now automatically inherits the max line length from the `Metrics/LineLength` cop.

**Comment by brodygov** [comment]
Dashboard deploys are failing on servers with these changes with a message about uglifier.
